Makefile

```diff
@@ -8,26 +8,7 @@ IMAGE_NAME = scorecard
 OUTPUT = output
 IGNORED_CI_TEST="E2E TEST:blob|E2E TEST:executable"
 
-# generate VERSION_LDFLAGS
-GIT_VERSION ?= $(shell git describe --tags --always --dirty)
-GIT_HASH ?= $(shell git rev-parse HEAD)
```

Comment:
`GIT_HASH` is also used here: https://github.com/ossf/scorecard/blob/main/Makefile#L158.

Reply:
oh oops.